drivers/ieee802154/ieee802154_cc13xx_cc26xx_subg.c

```diff
@@ -515,6 +527,26 @@ static int ieee802154_cc13xx_cc26xx_subg_stop(const struct device *dev)
 	return 0;
 }
 
+/**
+ * Stops the sub-GHz interface and yields the radio (tells RF module to power
+ * down).
+ */
+static int ieee802154_cc13xx_cc26xx_subg_stop_if(const struct device *dev)
```

Comment:
if `ieee802154_cc13xx_cc26xx_subg_stop()` is no longer used, let's rename this `ieee802154_cc13xx_cc26xx_subg_stop()` and delete the dead code / old implementation.

Reply:
It is required because `ieee802154_cc13xx_cc26xx_subg_stop` (the old one, without `_if`) only aborts the RX command, which is used through the module as utility function.